simulation_parameters/microbe_stage/biomes.json

```diff
@@ -1899,7 +1899,7 @@
         "oxygen": {
           "Amount": 0,
           "Density": 0,
-          "Ambient": 0.21
+          "Ambient": 0.22
```

Comment:
Shouldn't the max be kept as 21 as that's what's on Earth right now?

Reply:
From what I've read, the 21% represents more of an "average" throughout the entire atmosphere and not necessarily a max capacity. So local concentrations of atmosphere within patches can have more oxygen than 21% in them.